### ISSUE TITLE
Loss doesn't decrease

### ISSUE THREAD

**Issue by jackkwok** [tom-roddick/oft#6]
I am trying to train with the KITTI dataset by following the README.  Somehow, the total loss doesn't seem to decrease at all, even after training for 23 epochs (~6 hours).  How many epoch should we expect to see total loss to start decreasing?

==> Training epoch complete
score   : 2.3804e+01
position: 2.0489e+06
dimension: 4.0322e+05
angle   : 1.1116e+04
total   : 2.4632e+06

=== Beginning epoch 24 of 600 ===

I have Pytorch 1.3.0 if that matters.

**Comment by tjucwb** [comment]
@jackkwok  Have you finished training and does the training work? cauz I have more or less same question/problem as you

**Comment by tom-roddick** [comment]
Hi guys, sorry for the slow response. It turns out I had a line left in from a test I was running where only the score loss was being optimised.

If you delete the following line from the `compute_loss` function in `train.py`, the loss should start decreasing.
https://github.com/tom-roddick/oft/blob/2deed80f7c292c3cc4eaf8dfde544e28e32a0dde/train.py#L161-L162

I'll try and submit a new push shortly.

**Comment by qfwysw** [comment]
> I am trying to train with the KITTI dataset by following the README. Somehow, the total loss doesn't seem to decrease at all, even after training for 23 epochs (~6 hours). How many epoch should we expect to see total loss to start decreasing?
> 
> ==> Training epoch complete
> score : 2.3804e+01
> position: 2.0489e+06
> dimension: 4.0322e+05
> angle : 1.1116e+04
> total : 2.4632e+06
> 
> === Beginning epoch 24 of 600 ===
> 
> I have Pytorch 1.3.0 if that matters.

Hi jackkwok, do you have a problem with the validation set loss not decreasing?

**Comment by tom-roddick** [comment]
This issue should be fixed by 04f7383.

**Comment by IAMShashankk** [comment]
@jackkwok @qfwysw Did it worked for you? In my case loss is decreasing but very slow even after 10-12 hour training just from 2.5542e+04 to 2.2268e+04.
@tom-roddick Any inputs how can I fix it?
